editor.planx.uk/src/@planx/components/MapAndLabel/test/mocks/mockPayload.tsx

```diff
@@ -0,0 +1,23 @@
+export const mockSingleFeaturePayload = {
```

Comment:
Both here and in the above `mockTreeData` I'd suggest adding types. This would mean that when our component behaviour changes, TypeScript will force us to also update our tests and mocks.